mis_builder/views/mis_report_instance.xml

```diff
@@ -163,6 +163,7 @@
                                 <field
                                     name="company_id"
                                     groups="base.group_multi_company"
+                                    attrs="{'required': [('multi_company', '=', False)]}"
```

Comment:
Also, invisible if multi_company is True ?

Reply:
For a better UX, let's take the opportunity to move the multi_company checkbox before the company_id field.